packages/flame/pubspec.yaml

```diff
@@ -18,8 +18,8 @@ dev_dependencies:
   dartdoc: ^0.39.0
 
 environment:
-  sdk: ">=2.12.0-29.10.beta <3.0.0"
-  flutter: ">=1.24.0-10.2.pre"
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.0.0"
```

Comment:
Doesn't `>=1.24.0` works? This Would make people having to migrate to NNBD without risking breaking changes on flutter that may have come to 2.0.0
Unless we depend on something from 2.0.0

Reply:
If we move change to 1.24.0, we will need to decrease our dart version right?

Reply:
Nope, on PV the constraint is  as low as 1.6.0
https://github.com/fireslime/photo_view/blob/master/pubspec.yaml#L8

Reply:
hmmm, that seems a bit odd to me, but if that works, then I will change it